Monofoxe/Monofoxe.Engine/Drawing/BezierCurve.cs

```diff
@@ -0,0 +1,18 @@
+﻿using Microsoft.Xna.Framework;
+using Monofoxe.Engine.Utils;
+
+namespace Monofoxe.Engine.Drawing
+{
+	public class BezierCurve
+	{
+		public static void Draw(Vector2[] controlPoints, float interval = 0.01f)
```

Comment:
Add the autodoc comment describing what the method does and what the arguments are for.